_data/simple-icons.json

```
@@ -5386,6 +5386,11 @@
             "hex": "00D672",
             "source": "https://fizz.ca"
         },
+        {
+            "title": "Flash Flash Revolution",
+            "hex": "04a3c4",
```

I found `#28aad1` on both the header and "Download FFR" button on the home page.
```suggestion
            "hex": "28AAD1",
```

---

What color do you get on [FFR/'s Github page](https://github.com/flashflashrevolution)? I think I used that one instead.

Sorry, I'm now realizing that source means the location of the image, not the source of the webpage that the icon is affiliated by.

---

> What color do you get on [FFR's Github page](https://github.com/flashflashrevolution)? I think I used that one instead.

Not seeing any hex codes or SVGs here...